.ci/jobsDSL/templates/mbp.groovy.j2

```diff
@@ -0,0 +1,171 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Do you think we could move away from jinja and use a template class? Then everything will be groovy/java based?

Reply:
Do you mean a wrapper/builder? I do not like to hide this kind of templates, when you make it you hide the issues on the class you implemented, the documentation from jobDSL is no longer a valid place to find help because you have an opinionated way to generate the DSL.

Reply:
I understand the concerns, though if we use the same pattern then we can delegate the template to a specific class. In the end, we do nothing with the JJBB at the moment, for instance, and sometimes we need to change several places to apply the same configuration, so I see the benefits for using template classes in the future. This is not a blocker, but I guess we can discuss further in a follow up

Reply:
in that case, you can use a jobDSL template with groovy logic to generate those similar pipelines jobs, e.g.

```
[1, 2, 3 , 4].each{ it ->
  multibranchPipelineJob("it_${it}") {
....
  }
}
```